build.gradle

```diff
@@ -54,12 +54,19 @@ dependencies {
   implementation 'org.yaml:snakeyaml:1.29'
   implementation 'org.slf4j:slf4j-nop:1.7.30'
   implementation 'info.picocli:picocli:4.6.1'
+  implementation 'org.apache.httpcomponents:httpcore-nio:4.4.15'
+
 
   testImplementation 'junit:junit:4.12'
   testImplementation 'com.github.stefanbirkner:system-rules:1.19.0'
   testImplementation "org.mockito:mockito-core:3.+"
 }
 
+compileJava {
```

Comment:
We're building with JDK 1.8 in Travis, but these settings mean you can use any JDK.